Makefile.toml

```diff
@@ -85,19 +85,11 @@ cargo d
 [tasks.deadlink]
 script = ['''
 #!/usr/bin/env bash -eux
-mkdir -p target
-set +e
-# create deadlinks log file
-cargo deadlinks --check-http --ignore-fragments | tee target/cargo-deadlinks.out
-set -e
-# sort uniq for `cargo-deadlinks` output
-cat target/cargo-deadlinks.out | sort | uniq > target/cargo-deadlinks.sort
-# check diff for `deadlinks.expected`
-cat deadlinks.expected | sort | uniq > target/cargo-deadlinks.expected.sort
-diff -u target/cargo-deadlinks.expected.sort target/cargo-deadlinks.sort
-
+cargo doc
+for crate in $( ls target/doc | grep springql ); do
```

Comment:
I heard from @TaKO8Ki `cargo doc --no-deps` may be buggy so we might better to use just `cargo doc`.
Pretty good.

Could you consider to filter the target directories from crate names? (our crates might not include `springql` substring)

Reply:
You're right. I fixed it to get a list of crate names using `cargo-metadata`.